src/lib/gssapi/krb5/gssapiP_krb5.h

```diff
@@ -1425,4 +1425,14 @@ iakerb_gss_pseudo_random(OM_uint32 *minor_status, gss_ctx_id_t context_handle,
  * the format changes. */
 #define CRED_EXPORT_MAGIC "K5C1"
 
+#define GSS_KRB5_GET_CRED_IS_IMPERSONATOR_OID_LENGTH 11
+#define GSS_KRB5_GET_CRED_IS_IMPERSONATOR_OID  \
+    "\x2a\x86\x48\x86\xf7\x12\x01\x02\x02\x05\x0e"
+
+OM_uint32 KRB5_CALLCONV
+gss_krb5int_cred_is_impersonator(OM_uint32 *minor_status,
```

Comment:
I don't think you want KRB5_CALLCONV here.  I believe this is causing the appveyor build failure.